src/App.js

```diff
@@ -4,13 +4,28 @@ import Mockman from "mockman-js";
 import { Navbar } from "./components";
 import { Home, Products, Wishlist, Cart, Login, Signup } from "./pages";
 import { useAuth } from "./context/AuthContext";
+import { Toaster } from "react-hot-toast";
 function App() {
   const {
     state: { isLoggedIn },
   } = useAuth();
 
   return (
     <div className="App">
+      <Toaster
+        position="top-right"
+        reverseOrder={false}
+        toastOptions={{
+          // Define default options
+          className: "",
+          style: {
+            minWidth: "280px",
+          },
+          success: {
+            duration: 2000,
+          },
+        }}
+      />
```

Comment:
You can extract this into a component and then use it here so that the App.js looks clean!

Reply:
will refactor it. thank you